.devcontainer/devcontainer.json

```diff
@@ -32,6 +32,13 @@
 			"version": "os-provided"
 		}
 	},
+
+	// TODO: Improve to allow dynamic runArgs, see microsoft/vscode-remote-release#3972
+	// Uncomment this if GPU support is required
+	// "runArgs": [
+    //     "--gpus=all"
```

Comment:
Nit: can you align this comment

Reply:
That's so strange, in my working tree it looks correct:
![image](https://user-images.githubusercontent.com/22006760/197126024-e126d7c5-354a-4c5a-aeb6-a968677e23ae.png)

Let me try editing it on GitHub.

Reply:
Ok, that seemed to do the trick. :)